docs/element-definitions.md

```diff
@@ -1648,9 +1654,52 @@ series of bytes.
 }
 ```
 
-This reads, "The location starts at the 5th byte of the source file. It
-includes the 12 subsequent bytes including the starting one. Then it
-continues at the 21st byte for another 12 bytes."
+This reads, "The location starts at the 5th byte (the 2nd byte of line 3) of the source file".
```

Comment:
The previous text explained what multiple arrays inside source map mean. The new one doesn't, but the example still contains two arrays of numbers. Either the JSON should be adjusted or the explanation shouldn't be simpler.

Also, it seems there was a mistake in the JSON, because the previous text says something about 21st byte, but the JSON example is `[[4, 12], [4, 12]]`, i.e. it is addressing two identical blocks. It was probably supposed to be `[[4, 12], [21, 12]]`.